mcs_benchmark_data/cli/commands/_command.py

```diff
@@ -25,6 +26,7 @@ class _Command(ABC):
     def __init__(self):
         self._logger = logging.getLogger(self.__class__.__name__)
 
+    @classmethod
     def add_arguments(
         self, arg_parser: ArgParser, add_parent_args: FunctionType

```

Comment:
In a classmethod the first parameter is called 'cls' by convention. It's the class object being called (in this case the command subclass).